apps/root/src/pages/transfer/components/recipient-address/components/add-contact-modal/index.tsx

```diff
@@ -0,0 +1,227 @@
+import React from 'react';
+import { FormattedMessage, defineMessage, useIntl } from 'react-intl';
+import { Button, Grid, Modal, SuccessCircleIcon, TextField, Typography, colors } from 'ui-library';
+import useContactListService from '@hooks/useContactListService';
+import useStoredContactList from '@hooks/useStoredContactList';
+import useValidateAddress, { ValidationOutput } from '@hooks/useValidateAddress';
+import CenteredLoadingIndicator from '@common/components/centered-loading-indicator';
+import styled from 'styled-components';
+import ErrorCircleIcon from 'ui-library/src/icons/errorCircle';
+
+interface AddContactModalProps {
+  open: boolean;
+  setOpen: React.Dispatch<React.SetStateAction<boolean>>;
+  goBackToTransfer: () => void;
+}
+
+const StyledStatusTitle = styled(Typography).attrs({ variant: 'h5' })`
+  ${({ theme: { palette } }) => `
+  font-weight: bold;
+  color: ${colors[palette.mode].typography.typo1};
+  text-align: center;
+`}
+`;
+
+const StyledStatusDescription = styled(Typography).attrs({ variant: 'body1' })`
+  ${({ theme: { palette } }) => `
+  color: ${colors[palette.mode].typography.typo2}
+  text-align: center;
+`}
+`;
+
+const inputRegex = RegExp(/^[A-Fa-f0-9x]*$/);
+
+const PostContactStatusContent = ({
+  icon,
+  title,
+  description,
+  button,
+}: {
+  icon: React.ReactElement;
+  title: React.ReactElement;
+  description: React.ReactElement;
+  button: React.ReactElement;
+}) => (
+  <Grid container direction="column" rowGap={6}>
+    <Grid item xs={12}>
+      {icon}
+    </Grid>
+    <Grid container rowGap={2} direction="column">
```

Comment:
every child of a container grid needs to be an item. Take a look at https://mui.com/material-ui/react-grid/ since I'm seeing quite a bit of this